martin/src/pg/configurator.rs

```diff
@@ -232,12 +234,54 @@ impl PgBuilder {
     }
 }
 
+/// Try to find any ID column in a list of table columns (properties) that match one of the given `id_column` values.
+/// If found, modify `id_column` value on the table info.
+fn update_id_column(id: &str, inf: &mut TableInfo, auto_tables: &PgBuilderAuto) {
+    let Some(props) = inf.properties.as_mut() else { return };
+    let Some(try_columns) = &auto_tables.id_columns else { return };
+
+    for key in try_columns.iter() {
+        let (column, typ) = if let Some(typ) = props.get(key) {
+            (key, typ)
+        } else {
+            match find_kv_ignore_case(props, key) {
+                Ok(Some(result)) => {
+                    info!("For source {id} from table {}.{}, feature ID '{key}' was not found, but found '{result}' instead.", inf.schema, inf.table);
```

Comment:
the logging here and below might need to be cleaned up to make it consistent with the `pg::utilities` logging, and include the source ID in all, etc.